python-sdk/src/astro/databases/aws/redshift.py

```diff
@@ -384,3 +384,11 @@ def get_merge_initialization_query(parameters: tuple) -> str:
         for Redshift.
         """
         return "SELECT 1 + 1"
+
+    def openlineage_dataset_name(self, table: BaseTable = None):
+        conn = self.hook.get_connection(self.conn_id)
+        return f"{conn.schema}.{table.name}"
```

Comment:
wouldn't get qualified table method give you the same thing?

Reply:
qualified table method, consider schema name Table object schema field only so if I have schema in connection not Table then it won't be same

Reply:
Sure. Makes sense